cmd/nerdctl/compose_ps_linux_test.go

```diff
@@ -22,9 +22,89 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/containerd/nerdctl/pkg/tabutil"
 	"github.com/containerd/nerdctl/pkg/testutil"
+	"gotest.tools/v3/assert"
 )
 
+func TestComposePs(t *testing.T) {
+	testutil.DockerIncompatible(t)
```

Comment:
Why?

Reply:
removed, and the DockerIncompatible() in TestComposePsJSON as well.